packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -74,9 +74,7 @@ const GroupAction = (props: GroupActionProps) => {
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [isRendering, setIsRendering] = useState<boolean>(false);
   const [currentPropertyList, setCurrentPropertyList] = useState<PropertyRecord[]>([]);
-  const [queryBuilder, setQueryBuilder] = useState<QueryBuilder>(
-    new QueryBuilder(undefined),
-  );
+  const [queryBuilder, setQueryBuilder] = useState<QueryBuilder>();
```

Comment:
`QueryBuilder` is very specific to `GroupQueryBuilderContainer`. Move it out of `GroupAction`.

Reply:
Transferred to `GroupQueryBuilderContainer`